styx-cli/src/main/java/com/spotify/styx/cli/PlainCliOutput.java

```diff
@@ -120,7 +120,7 @@ public void printMessage(String message) {
   public void printWorkflow(Workflow wf, WorkflowState state) {
     System.out.println(Joiner.on(' ').join(
         wf.componentId(),
-        wf.id(),
+        wf.id().id(),
```

Comment:
This fix breaks backward compatibility a bit.

Reply:
I'll be very surprised if anyone was relying on this

Reply:
Yeah, I didn't find anything.